osf/metrics.py

```diff
@@ -275,17 +275,21 @@ def get_current_user_metrics(cls, institution) -> list:
         """
         last_record_time = cls.get_recent_datetime(institution)
 
-        return cls.filter_institution(
+        search = cls.filter_institution(
             institution
-        ).sort(
-            'timestamp'
         ).filter(
             'range',
             timestamp={
                 'gte': last_record_time
             }
+        ).sort(
+            'user_id'
         )
+        search.update_from_dict({
+            'size': 1000
```

Comment:
Should use `api.base.settings.MAX_SIZE_OF_ES_QUERY` for this

Reply:
Ah! Didn't know that was a setting!